source/modules/src/data/sql/database-modules/nature_areas/base_21.sql

```diff
@@ -1,8 +1,8 @@
-BEGIN; SELECT system.load_table('nature.countries', '{data_folder}/temp/temp_countries_20150721.txt', FALSE); COMMIT;
-BEGIN; SELECT system.load_table('nature.authorities', '{data_folder}/public/authorities_20191105.txt', FALSE); COMMIT;
+BEGIN; SELECT system.load_table('nature.countries', '{data_folder}/common/nature_areas/21/nature.countries_20150721.txt', FALSE); COMMIT;
```

Comment:
have the temp countries become permanent?

Reply:
Temp since 2015 I think is pretty permanent ;) And this file has already been used in many releases.